migrations/124_instance_questions__used_for_grade__add.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE instance_questions ADD COLUMN IF NOT EXISTS used_for_grade BOOLEAN;
```

Comment:
Should we make this `NOT NULL` and `DEFAULT FALSE`?

Reply:
If I understand correctly, doing so would populate the new column with `FALSE` everywhere. I don't think this is what we want.

Leaving this migration as is allows us to distinguish between cases in which instance questions have or have not been used (`TRUE` or `FALSE`) and cases in which - because they occurred prior to this PR - we don't know (`NULL`).

Does that make sense?

Reply:
Yep, agreed.